client/state/importer-nux/utils.js

```diff
@@ -0,0 +1,13 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { memoize } from 'lodash';
+
+export const normalizeImportUrl = memoize( url =>
```

Comment:
Ohh, actually, before we merge I think it might be worth just giving url a default value of `''`. That would guard against type errors where url is `undefined` or `null`. I doubt we'd ever pass in `undefined` but just-in-case. What do you think?